### DISCUSSION TITLE
Updating Null Values

### DISCUSSION THREAD

**Discussion by vibhav-zf** [RobertCraigie/prisma-client-py#769] [Q&A] [answered]
Hi, I was trying to update a column to `NULL` but couldn't find anything in the docs and the prisma official docs are for js where `null` and `undefined` are different.

I tried with `None` but that didn't do anything (I assume it corresponds to undefined).
How can I update a value to `NULL`?

Thanks!

**Comment by royalbhati** [comment]
+1

**Comment by zfundsdushyant** [comment]
Facing the same issue

**Comment by RobertCraigie** [comment, marked as answer]
Hey @vibhav-zf @royalbhati @zfundsdushyant, can you share a code sample that isn't working for you?

This should be possible by just passing `None`, see this test case for an example: https://github.com/RobertCraigie/prisma-client-py/blob/805136e2ec60ab6393ae09facb46e1890c4c17b2/databases/tests/test_update.py#L154-L162

**Reply by vibhav-zf** [reply]
I tried this and it did not work - I must be doing something wrong, I just tried this again and it does, thanks.
